src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,12 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
```

Comment:
you can use destructuring for comment

Reply:
<img width="637" alt="Screenshot 2024-07-23 at 21 32 08" src="https://github.com/user-attachments/assets/d6a50cc6-5960-49ad-9b8b-864a54853a66">


I think that I had it

Reply:
or I just don't understand what you mean

Reply:
Feel free to ask for help in the chat https://uk.javascript.info/destructuring-assignment

Reply:
this is a js documentation

Reply:
of course, because React is a js library))0)